pkg/pod/entrypoint_test.go

```diff
@@ -872,3 +872,64 @@ func TestStopSidecars(t *testing.T) {
 		})
 	}
 }
+
+func TestIsSidecarStatusRunning(t *testing.T) {
+	type args struct {
+		tr *v1beta1.TaskRun
+	}
+	tests := []struct {
+		name string
+		args args
```

Comment:
please add a taskrun here directly

```suggestion
		tr *v1beta1.TaskRun
```